drivers/exec/driver.go

```diff
@@ -88,6 +93,9 @@ type Driver struct {
 	// event can be broadcast to all callers
 	eventer *eventer.Eventer
 
+	// config is the driver configuration set by the SetConfig RPC
+	config *Config
```

Comment:
`d.config` might be nil in tests - hence one test failure.  You may consider avoid using the pointer here.

```suggestion
	config Config
```